crates/masonry/src/event_loop_runner.rs

```diff
@@ -307,22 +307,35 @@ impl MainState<'_> {
 pub(crate) fn try_init_tracing() -> Result<(), SetGlobalDefaultError> {
     #[cfg(not(target_arch = "wasm32"))]
     {
+        use time::macros::format_description;
         use tracing_subscriber::filter::LevelFilter;
+        use tracing_subscriber::fmt::time::UtcTime;
         use tracing_subscriber::prelude::*;
         use tracing_subscriber::EnvFilter;
 
+        // Default level is DEBUG in --dev, INFO in --release
+        // DEBUG should print a few logs per low-density event.
+        // INFO should only print logs for noteworthy things.
         let default_level = if cfg!(debug_assertions) {
             LevelFilter::DEBUG
         } else {
             LevelFilter::INFO
         };
+        // Use EnvFilter to allow the user to override the log level without recompiling.
+        // TODO - Print error message if the env var is incorrectly formatted.
         let env_filter = EnvFilter::builder()
             .with_default_directive(default_level.into())
             .with_env_var("RUST_LOG")
             .from_env_lossy();
+        // This format is more concise than even the 'Compact' default:
+        // - We print the time without the date (GUI apps usually run for very short periods).
+        // - We print the time with seconds precision (we really don't need microseconds=.
```

Comment:
Milliseconds would surely be useful?

Reply:
In GUI? I don't see it. We don't do multi-threading yet, and we deal very little with network events. In my experience there hasn't been one case where the millisecond count was helpful.

The coarse time at least helps me get some context, eg "Ok, this happened 3 minutes ago when I was doing X, so this is probably Y".

Reply:
I was under the impression that we should think about GUI as a soft-realtime system, and so it is important to be following e.g. how long things take, and milliseconds is the right scale for that
I know I found the timestamps very useful in my Vello work.

But maybe our conceptions of what these logs are for are misaligned

Reply:
We can still add that later, when it makes sense I guess. I do lean to also finding milliseconds useful though.

Reply:
Part of the reason I'm moving this way, is that I plan to produce multiple logs at once, as outlined in #250. The console logs will be the least detailed ones; the logs written to a temp file will have the full detail level.